src/ontology/cl.Makefile

```diff
@@ -175,10 +175,10 @@ test: $(REPORTDIR)/taxon-constraint-check.txt
 # ----------------------------------------
 
 .PHONY: obocheck
-obocheck: $(SRC)
+obocheck: $(SRC) | all_robot_plugins
 	$(ROBOT) merge -i $(SRC) \
 		 remove --base-iri $(URIBASE)/CL_ --axioms external --trim false \
-		 convert -f obo --check false -o $(TMPDIR)/cl-check.obo
+		 uberon:obo-export --merge-comments --obo-output $(TMPDIR)/cl-check.obo
```

Comment:
oh my woooooooord - I would love a generic OBO "fixer" tool that does things like merging comments and dropping duplicate labels! This _should_ really be in ROBOT but hey! Plugin is just as well!

Reply:
I will certainly consider upstreaming that to ROBOT at some point. At the very least for the `--strip-owl-axioms` option (removing the untranslatable axioms), since (1) the ODK already does that systematically, so presumably everybody is already fine with that behaviour, and (2) the OWLAPI already provides the required option to enable that behaviour, it’s just that ROBOT is not using it.